config.json

```diff
@@ -934,8 +934,15 @@
         "name": "Kindergarten Garden",
         "uuid": "04bde625-e363-4d8b-880f-db7bf86286eb",
         "practices": ["symbols"],
-        "prerequisites": ["strings", "symbols", "enumerable"],
-        "difficulty": 3,
+        "prerequisites": [
+          "strings",
+          "symbols",
+          "enumerable",
+          "dynamic-programming",
```

Comment:
Are we sure about this one?  What hints would you write concerning this if you were to write some mentor notes about dynamic programming concerning this exercise?